src/example-types/metricGroups/numberRangesGroupStats.js

```diff
@@ -11,8 +11,8 @@ let buildGroupQuery = ({ field, ranges }, children, groupingType) => ({
 })
 
 let drilldown = ({ field, drilldown }) => {
-  let [gte, lte] = _.split('-', drilldown)
-  return { range: { [field]: { gte, lte } } }
+  let [gte, lt] = _.split('-', drilldown)
+  return { range: { [field]: { gte, lt } } }
```

Comment:
I was thinking about this more today, and I'm not sure it's right. For example, on dates, you might say 1/1/2021 - 12/31/2021 to represent 2021. I think this is really just a Bradley question on whether it should lte or lt (or maybe even exposed in the UI somehow)